step-certificates/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: step-certificates
-version: 1.15.17
+version: 1.15.18
```

Comment:
One note on the version bump.

Since this PR changed the expected format of several of the values parameters, we may want to bump a minor version instead of a patch version if anyone has taken a dependency on 1.15.17.